From 5de43723db7bc86bce09c5c491656f1e7ddfa953 Mon Sep 17 00:00:00 2001
From: scx <scx.mail@gmail.com>
Date: Sat, 18 May 2019 19:25:00 +0200
Subject: [PATCH] Fix AppData file

 * Add `<translation>` tag
 * Remove invalid `<i>` tag
 * `<li>` cannot end in '.'
---
 data/qalculate-gtk.appdata.xml.in | 45 ++++++++++++++++---------------
 1 file changed, 23 insertions(+), 22 deletions(-)

diff --git a/data/qalculate-gtk.appdata.xml.in b/data/qalculate-gtk.appdata.xml.in
index c1e74a7..8ce9274 100644
--- a/data/qalculate-gtk.appdata.xml.in
+++ b/data/qalculate-gtk.appdata.xml.in
@@ -43,23 +43,24 @@
  <provides>
   <binary>qalculate-gtk</binary>
  </provides>
+ <translation type="gettext">qalculate-gtk</translation>
  <releases>
   <release version="3.1.0" date="2019-04-15">
    <description>
     <p>Changes:</p>
      <ul>
-      <li>Calculation of Si(), Ci(), Shi(), and Chi().</li>
-      <li>Convert trigonometric arguments to selected angle unit, and hide angle unit in result.</li>
-      <li>Hide optional arguments, with default value, in output.</li>
-      <li>Add optional variable value argument to diff(), for calculation of derivative for a specific <i>x</i> value.</li>
-      <li>Improved sinc() support.</li>
-      <li>Improved output with show ending zeroes activated.</li>
-      <li>Fix ignore system language on Windows.</li>
-      <li>Fix atomic unit of velocity.</li>
-      <li>Fix and improve keypad menus, and add dynamic convert to menu to result popup menu (GUI).</li>
-      <li>Fix compilation errors because of erroneous limits.h inclusion.</li>
-      <li>Fix "factorized result is wrong" error.</li>
-      <li>Minor bug fixes and feature improvements.</li>
+      <li>Calculation of Si(), Ci(), Shi(), and Chi()</li>
+      <li>Convert trigonometric arguments to selected angle unit, and hide angle unit in result</li>
+      <li>Hide optional arguments, with default value, in output</li>
+      <li>Add optional variable value argument to diff(), for calculation of derivative for a specific x value</li>
+      <li>Improved sinc() support</li>
+      <li>Improved output with show ending zeroes activated</li>
+      <li>Fix ignore system language on Windows</li>
+      <li>Fix atomic unit of velocity</li>
+      <li>Fix and improve keypad menus, and add dynamic convert to menu to result popup menu (GUI)</li>
+      <li>Fix compilation errors because of erroneous limits.h inclusion</li>
+      <li>Fix "factorized result is wrong" error</li>
+      <li>Minor bug fixes and feature improvements</li>
      </ul>
    </description>
   </release>
@@ -67,16 +68,16 @@
    <description>
     <p>Changes:</p>
      <ul>
-      <li>Implement 2019 redefinition of SI base units (primarily affects physical constants and will not take effect until 20 May).</li>
-      <li>Uncertainty propagation improvements/fixes.</li>
-      <li>Separate use of interval arithmetic, to keep track of precision changes with approximate calculations, from selection of interval calculation algorithm (by default both interval arithmetic and uncertainty propagation using variance formula are now activated).</li>
-      <li>uncertainty() function (can specify relative uncertainty of value).</li>
-      <li>Improved polynomial gcd.</li>
-      <li>Option to ignore system language (locale).</li>
-      <li>Option to use two's complement also for hexadecimal numbers.</li>
-      <li>Switch to exact decimal fractions (fixed) from decimal fractions mode (and vice versa) when exact mode is toggled.</li>
-      <li>Restore lost signal handlers in the conversion GUI (fixes search, popup menu, etc.).</li>
-      <li>Bug fixes and minor feature improvements.</li>
+      <li>Implement 2019 redefinition of SI base units (primarily affects physical constants and will not take effect until 20 May)</li>
+      <li>Uncertainty propagation improvements/fixes</li>
+      <li>Separate use of interval arithmetic, to keep track of precision changes with approximate calculations, from selection of interval calculation algorithm (by default both interval arithmetic and uncertainty propagation using variance formula are now activated)</li>
+      <li>uncertainty() function (can specify relative uncertainty of value)</li>
+      <li>Improved polynomial gcd</li>
+      <li>Option to ignore system language (locale)</li>
+      <li>Option to use two's complement also for hexadecimal numbers</li>
+      <li>Switch to exact decimal fractions (fixed) from decimal fractions mode (and vice versa) when exact mode is toggled</li>
+      <li>Restore lost signal handlers in the conversion GUI (fixes search, popup menu, etc.)</li>
+      <li>Bug fixes and minor feature improvements</li>
      </ul>
    </description>
   </release>
